lib/vapor/store/supervisor.ex

```diff
@@ -6,14 +6,24 @@ defmodule Vapor.Store.Supervisor do
 
   use Supervisor
 
+  alias Vapor.{
+    Store,
```

Comment:
We don't use this alias because on line 26 we have the full module name of `Vapor.Store`.

Reply:
Fixed up